QuizApp/View/Quiz/QuizViewController.swift

```diff
@@ -17,12 +17,16 @@ class QuizViewController: UIViewController {
     private var question: Question!
     private var questionNum: Int = 0
     private var correct: Int = 0
+    private var start: DispatchTime!
+    private var presenter: QuizPresenter!
     
     convenience init(router: AppRouter, quiz: Quiz) {
         self.init()
         self.router = router
         self.quiz = quiz
         question = quiz.questions[questionNum]
+        presenter = QuizPresenter(router: router)
```

Comment:
Inject through the initializer, don't create it here. Keep the view controller as slim as you possibly can.